api/swagger.yml

```diff
@@ -1134,6 +1144,24 @@ paths:
           default:
             $ref: "#/components/responses/ServerError"
 
```

Comment:
we said that this is called from a place where the user forgot this password. so this request can't be authenticated. you need to specify in the swagger not to check this route.

Reply:
I'm afraid I still did not understand, are we are referring to server error in the case of forgot password? If that is the case, why shouldn't we check server error (this can happen if the token generation failed for instance)?

Reply:
Thank for pointing this out, missed this one, fixed now